### PR TITLE
add custom containers and elements to lightbox

### PR DESCRIPTION
**Pull request by deoxxa**

### CODE REVIEW

File: packages/lightbox/README.md [hunk 1]
```diff
@@ -50,3 +50,7 @@ Array of assets to render in the Lightbox. They should be objects with the follo
 * `media`, the MIME type of the asset.
 * `title`, the title or file name of the asset.
 * `path`, the URL of the asset itself.
+* `container`, a React class implementing a container for this asset (optional,
+	negates the requirement for `media`).
+* `element`, a React element to use directly instead of instantiating one
+	internally (optional, negates the requirement for `media` and `path`).
```
Comment (grapegravity) [inline packages/lightbox/README.md]: Could you update `propTypes` with these two?

File: packages/lightbox/example.jsx [hunk 2]
```diff
@@ -36,6 +48,15 @@ const assets = [
     title: 'Unpreviewable or Unrecognised Type',
     path: require('./images/forest.jpg'),
   },
+  {
+    title: 'Custom container',
+    container: CustomContainer,
+    path: 'not a real path!',
+  },
```
Comment (grapegravity) [inline packages/lightbox/example.jsx]: What’s the rationale for this method over registering it as a container?

Comment (deoxxa) [inline packages/lightbox/example.jsx]: Avoidance of global state when not necessary, mostly. If I don't have a specific reason to make something global, I don't want to be forced to do so.

Comment (grapegravity) [inline packages/lightbox/example.jsx]: Mm, fair. I wonder if it’d be better to make that not global, then… Not that that’s part of this PR!

**Pull request merged**